### DISCUSSION TITLE
Bulk Edit Relationships

### DISCUSSION THREAD

**Discussion by CovantaDataManagement** [Budibase/budibase#4026] [Help]
I've gotten comfortable using your API's to insert/update/delete.  I could use a little guidance on something.

Through budibase, I have imported our employees into an employees table and their respective users records into the users table.  Both share a field called employee id and email_address.  The timecard submission application relies on a relationship between the employees and users table.  I have set several up by hand.  That said, I have 3,000 more to go.  What would the body look like to establish the relationship? 

Using a POST - I was able to submit this which came back successful.  However, it did not actually update anything:
{
    "rows": [
        {
            "rows": [
                {
                    "_id": "ro_ta_users_us_0e85b0b3c59147daa76e4e9ba671f800",
                    "_rev": "1-6efb903eb93732cc6a581f92bbf0f059",
                    "createdAt": 1640037172459,
                    "email": "**@covanta.com",
                    "firstName": "**",
                    "lastName": "**",
                    "status": "active",
                    "empId": "72125",
                    "tenantId": "default",
                    "roleId": "role_d4860c5a71a6494da753e039e17d944c",
                    "tableId": "ta_users",
                    "joinEmail": [
                        "ro_ta_7b0559377b674b1cab8a1bb1c2bcd730_3639bbf9164544c2ae55bb1da5b636d1"
                    ],
                    "EMPLOYEEDATA": [
                        "ro_ta_7b0559377b674b1cab8a1bb1c2bcd730_3639bbf9164544c2ae55bb1da5b636d1"
                    ],
                    "EMPLOYEEID": "72125"
                }
            ]
        }
    ]
}

**Comment by mjashanks** [comment]
try...

```json
{
  "_id": "ro_ta_users_us_0e85b0b3c59147daa76e4e9ba671f800",
  "_rev": "1-6efb903eb93732cc6a581f92bbf0f059",
  "createdAt": 1640037172459,
  "email": "email@covanta.com",
  "firstName": "First Name",
  "lastName": "Last Name",
  "status": "active",
  "empId": "72125",
  "tenantId": "default",
  "roleId": "role_d4860c5a71a6494da753e039e17d944c",
  "tableId": "ta_users",
  "joinEmail": [
    "ro_ta_7b0559377b674b1cab8a1bb1c2bcd730_3639bbf9164544c2ae55bb1da5b636d1"
  ],
  "EMPLOYEEDATA": [
    "ro_ta_7b0559377b674b1cab8a1bb1c2bcd730_3639bbf9164544c2ae55bb1da5b636d1"
  ],
  "EMPLOYEEID": "72125"
}
```

Note that I redacted names and email from the request
